.golangci.yml

```diff
@@ -1,8 +1,11 @@
 ---
 run:
   timeout: 4m
-  skip-dirs:
+
+issues:
+  exclude-dirs:
     - internal
```

Comment:
This was there before, but curious: why are we not linting `internal`? `internal` is just a visibility thing, we should still keep the code to the same standard no?

Reply:
actually removed. looks like this directory doesn't exists